src/uhh.cpp

```diff
@@ -36,7 +36,7 @@ int main(int argc, char** argv) {
         cmd
     };
 
-    Uhh* uhh = new Uhh(opts);
```

Comment:
So I did not realize this was possible.  You can simply "new" on the stack?  That is really neet.  Like, really really neet.